pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/applicationconfiguration.go

```diff
@@ -252,8 +248,8 @@ func (r *OAMApplicationReconciler) Reconcile(req reconcile.Request) (result reco
 	// make sure this is the last functional defer function to be called
 	defer func() {
 		// Make sure if error occurs, reconcile will not happen too frequency
-		if returnErr != nil && result.RequeueAfter < shortWait {
-			result.RequeueAfter = shortWait
+		if returnErr != nil {
+			result.RequeueAfter = 0
```

Comment:
why don't need requeue here?

Reply:
because controller runtime has auto requeue for any error with ratelimit